src/renderer/components/dex/OrderHistory.vue

```diff
@@ -203,16 +203,51 @@ export default {
 
 
   .body {
-    height: calc(100% - 73px);
-    overflow-y: auto;
     padding: $space;
+    height: calc(100% - 73px);
 
     .order-history-table {
       @extend %dex-table;
 
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+
+      thead {
+        display: block;
+      }
+
+      tbody {
+        display: block;
+        height: 6rem;
+        overflow-y: auto;
+
+        .status > div {
```

Comment:
![image](https://user-images.githubusercontent.com/10226077/44007717-9021a136-9e57-11e8-8847-6e7ee2340c01.png)

This status div looks to be different height then the rest of the columns. Looks like the SVG icon is pushing it out.

Reply:
![image](https://user-images.githubusercontent.com/10226077/44065516-5ba69b56-9f28-11e8-9c6b-769ce4614218.png)

It looks like the height of tbody is set to a constant 6rem. This doesn't seem right. I would imagine we want tbody 'flex' to the height of the component.

Reply:
tbody needs a fixed height otherwise it won't produce a scrollbar.

Reply:
It can probably go a little higher, but I don't think the height of this area for the table will change will it?

Reply:
The height you are using isn't using the full area; and the height will certainly change if they resize the window.